tests/integration/testlib/nmplugin.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from contextlib import contextmanager
+from subprocess import CalledProcessError
+
+from . import cmdlib
+
+
+@contextmanager
+def disable_nm_plugin(plugin):
+    try:
+        ret, rpm_output, _ = cmdlib.exec_cmd(
+            ("rpm", "-ql", f"NetworkManager-{plugin}"), check_returncode=True
+        )
```

Comment:
The except CalledProcessError should move here to guard the rpm -ql.

Reply:
Hmm not sure. If this command fails it means the plugin is not installed. In that case, we should proceed with the test directly without unmounting or restarting NM. What do you think? Thanks!

Reply:
I fixed this, thanks!